www/docs/en/6.x/config_ref/images.md

```diff
@@ -41,6 +41,7 @@ platform      | *Optional* <br/> Target platform
 width         | *Optional* <br/> Icon width in pixels
 height        | *Optional* <br/> Icon height in pixels
 density       | *Optional* <br/> Specified icon density (Android Specific)
+target        | *Optional* <br/> Destination filename for the image file and all its' MRT companions (Windows Specific)
```

Comment:
Thanks for advice, i've made similar change for `density` attribute as well (`==Android==`)